morph_tool/__init__.py

```diff
@@ -5,7 +5,6 @@
 from morph_tool.converter import convert
 from morph_tool.morphio_diff import diff
 from morph_tool.spatial import point_to_section_segment
-from morph_tool.version import __version__
```

Comment:
`__version__` is also used in doc/source/conf.py, I think there is a snippet from setuptools_scm specifically for sphinx.

Reply:
it now uses
```python
 version = get_distribution('morph_tool').version
```